packages/elements/src/clock/index.ts

```diff
@@ -601,10 +666,27 @@ export class Clock extends ResponsiveElement {
    */
   protected firstUpdated (changedProperties: PropertyValues): void {
     super.firstUpdated(changedProperties);
-    this.renderRoot.addEventListener('keydown', (event) => this.onKeydown(event as KeyboardEvent));
+    this.addEventListener('keydown', (event) => this.onKeydown(event));
     this.renderRoot.addEventListener('tapstart', (event) => this.onTapStart(event as TapEvent));
   }
 
+  protected update (changedProperties: PropertyValues): void {
+    super.update(changedProperties);
+
+    if (changedProperties.has('interactive')) {
+      if (this.interactive) {
+        this.tabIndex = 0;
```

Comment:
I am not sure you need to changed tabIndex on the host if children have own tabindex. Furthermore, why not to make it the same way as time-picker, where a `role=status` element is created to announce values?

Reply:
It prefers to use `aria-valuetext` instead of having an `aria-live` or `role=status` element. Because having `aria-live` or `role=status` element inside `role=spinbutton` isn't work for screen reader. I have proven with Goh that it doesn't work.
To make it work, it should place in the same level of dom.